File: patch_app.py
with open('src/App.tsx', 'r') as f:
    content = f.read()

if 'AdminProfessionalDashboardScreen' not in content:
    content = content.replace("const AdminDashboardScreen = lazy(() => import('./components/AdminDashboardScreen'));", "const AdminDashboardScreen = lazy(() => import('./components/AdminDashboardScreen'));\nconst AdminProfessionalDashboardScreen = lazy(() => import('./components/AdminProfessionalDashboardScreen'));")
    
    content = content.replace('<Route path="/admin-dashboard"', '<Route path="/admin/professional-dashboard" element={<ProtectedRoute role="admin"><AdminProfessionalDashboardScreen /></ProtectedRoute>} />\n              <Route path="/admin-dashboard"')
    
    with open('src/App.tsx', 'w') as f:
        f.write(content)

File: patch_app.mjs
import fs from 'fs';

let content = fs.readFileSync('src/App.tsx', 'utf8');
content = content.replace("import CentreStaffSelectionScreen from './components/CentreStaffSelectionScreen';", "import CentreStaffSelectionScreen from './components/CentreStaffSelectionScreen';\nimport CentreSalaryScreen from './components/CentreSalaryScreen';");
content = content.replace('<Route path="/centre/:centerId/staff" element={<CentreStaffSelectionScreen />} />', '<Route path="/centre/:centerId/staff" element={<CentreStaffSelectionScreen />} />\n          <Route path="/centre/:centerId/salary" element={<CentreSalaryScreen />} />');

fs.writeFileSync('src/App.tsx', content);
